.buildbot.sh

```diff
@@ -12,7 +12,7 @@ echo "Checking clang-format..."
 CLANG_FORMAT=$CHERI_DIR/morello-sdk/bin/clang-format
 find . -iname "*.c" -o -iname "*.h" -o -iname "*.cpp" -o -iname "*.hpp" | xargs "$CLANG_FORMAT" --dry-run -Werror
 
-echo "Checking that all the purecap examples build for all platforms..."
+echo "Checking that all examples build for all platforms..."
```

Comment:
I'm not sure if removing purecap here is fine, since this block should only be building purecap examples.

Reply:
They're cleaned up a bit in [73485f1](https://github.com/capablevms/cheri-examples/pull/96/commits/73485f1e63609c6be691b323fbb89dfb5b01b056).